### PR TITLE
[deprecation round2] LPS-103868 com.liferay.exportimport.kernel

### PR DESCRIPTION
**Pull request by dantewang**
cc @yuhai

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test

**Comment by yuhai** [comment]
@tinatian, 
I checked modules-semantic-versioning-jdk8, modules-unit-jdk8, modules-integration-mysql57-jdk8 and DLFileEntryIndexerLocalizedContentTest, these errors are unrelated with the fix.

**Comment by tinatian** [comment]
@dantewang @yuhai 
I did not include two commits, please see my comments.
Please see https://github.com/shuyangzhou/liferay-portal/pull/8488.

Tina.

File: portal-kernel/src/com/liferay/exportimport/kernel/lar/BasePortletDataHandler.java [hunk 1]
```diff
@@ -477,50 +477,6 @@ public void prepareManifestSummary(
 		}
 	}
 
-	/**
```
Comment (tinatian) [inline portal-kernel/src/com/liferay/exportimport/kernel/lar/BasePortletDataHandler.java:480]: @dantewang @yuhai 
This deprecation is weird according to the commit history, only implementation was deprecated, but not the interface method, also, this deprecation was done in a ticket which is dealing with exception handling, and all other logic is related to exception handling except this deprecation. Thus I can not tell if this was done on purpose or by accident. 

Let's hold on for this one, and move on with others.

Tina

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/PortletDataContextImpl.java [hunk 2]
```diff
@@ -948,15 +948,6 @@ public Object getNewPrimaryKey(String className, Object newPrimaryKey) {
 		return _newPrimaryKeysMaps;
 	}
 
-	/**
```
Comment (tinatian) [inline modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/PortletDataContextImpl.java:981]: Hi @dantewang @yuhai ,
This actually changed the info we dump into file. Even though we are sure on one is using it, it is still risky. 
Considering we are going to replace the whole staging with change tracking soon, I would like to hold on for this one and play safe for this round of deprecation removal.

Thanks.
Tina.

**Pull request closed without merging**